### PR TITLE
Add a new API to take Persisted Results

### PR DESCRIPTION
**Pull request by CharlleDaniel**
This PR is able to: 
- Add a new API to take persited results about compliance policies.

### CODE REVIEW

File: lib/xclarity_client/xclarity_base.rb [hunk 1]
```diff
@@ -84,5 +96,33 @@ def do_delete (uri="")
...
+        request.headers['Content-Type'] = 'application/json'
+        request.body = { :UserId => conf.username,
+                         :password => conf.password }.to_json
+      end
+      raise Faraday::Error::ConnectionFailed unless @response.success?
+    rescue => error
+      translation_error(error)
+    end
+
+    def translation_error(error)
```
Comment (douglasgabriel) [inline lib/xclarity_client/xclarity_base.rb]: maybe you want to call this method `translate_error`

File: lib/xclarity_client/xclarity_base.rb [hunk 2]
```diff
@@ -84,5 +96,33 @@ def do_delete (uri="")
...
+      cause = error.cause.to_s.downcase
+      if cause.include?('connection refused')
+        raise XClarityClient::Error::ConnectionRefused
+      elsif cause.include?('name or service not known')
+        raise XClarityClient::Error::HostnameUnknown
+      elsif @response.nil?
```
Comment (douglasgabriel) [inline lib/xclarity_client/xclarity_base.rb]: For a more [pure](https://en.wikipedia.org/wiki/Pure_function) approach, I think that will be a good idea to receive the  `@response` as a param

**Comment by douglasgabriel** [comment]
Can you add some tests for this new endpoint?

File: lib/xclarity_client/xclarity_base.rb (hunk 1 above)
Comment (CharlleDaniel) [reply lib/xclarity_client/xclarity_base.rb]: done

**Comment by CharlleDaniel** [comment]
@douglasgabriel  I will add some tests, thanks :+1:

**Comment by douglasgabriel** [comment]
LGTM

**Comment by CharlleDaniel** [comment]
@rodneyhbrown7 Could you review this PR?

**Comment by CharlleDaniel** [comment]
@felipedf Could you review this PR, please?

**Comment by felipedf** [comment]
LGTM  :+1:

**Pull request merged**